lms/djangoapps/teacher_dashboard/tests/test_views.py

```diff
@@ -65,15 +66,26 @@ def make_coach(self):
         Create coach user.
         """
         role = CourseCcxCoachRole(self.course.id)
-        role.add_users(self.coach)
+        role.add_users(self.user)
 
     def make_ccx(self):
         """
         Create ccx.
         """
-        ccx = CcxFactory(course_id=self.course.id, coach=self.coach)
+        ccx = CcxFactory(course_id=self.course.id, coach=self.user)
         return ccx
 
+    def test_student_cannot_see_teacher_dashboard(self):
+        """
```

Comment:
> As we decide before, should we write BDD for integration tests?

Yes, in case of sophisticated tests that hard to understand.

In this case, I can simply add `send request and verify status code`, but I think code is pretty readable.